### PR TITLE
Users view

### LINKED ISSUE #37: Create Manage Users pages
Creating a page for administrators to change user roles and delete users

### PR DESCRIPTION
**Pull request by barBen**
#37

### CODE REVIEW

**Review by bsuro10 — CHANGES_REQUESTED** [review]
When you want to delete a user,
the password is plain text and not type=password

After deleting, return to the list of users instead of home screen

**Review by origranot — DISMISSED** [review]
Good job!

**Review by ghost — DISMISSED** [review]

File: DGN/Views/Users/Edit.cshtml [hunk 1]
```diff
@@ -4,59 +4,65 @@
     ViewData["Title"] = "Edit Profile";
```
Comment (ghost) [inline DGN/Views/Users/Edit.cshtml]: There should not be Edit for normal edit, only for edit as admin.
Edit for the user itself is done from the profile, this file should be deleted.

File: DGN/Views/Users/EditAsAdmin.cshtml [hunk 2]
```diff
@@ -4,61 +4,67 @@
     ViewData["Title"] = "Edit User";
 }
```
Comment (ghost) [inline DGN/Views/Users/EditAsAdmin.cshtml:6]: like you did to the register to test the data before sending it to the the server this needs to be on the edit as admin too. (its just the first name, last name to start with capital letter and the email). correct me if i'm wrong @bsuro10

**Review by ghost — CHANGES_REQUESTED** [review]

File: DGN/Views/Users/EditAsAdmin.cshtml (hunk 2 above)
Comment (barBen) [reply DGN/Views/Users/EditAsAdmin.cshtml:6]: Maybe I did not understand, but if one of them is not as required (First name with capital letter, etc..) you get an error and cant submit the form..

**Review by ghost — APPROVED** [review]

**Review by origranot — APPROVED** [review]

**Review by bsuro10 — CHANGES_REQUESTED** [review]
- When editing a user, his role change to Client (Even if he was admin or author)
- The error message on First Name is not good: (Need to change in Modal to include that name must have 2 chars at least)
Bug0
- When you delete a user - return to the list of users
- You disabled the "Register" button if the user didn't fill the required fields, but you need to add something to tell him what is required and what is not, add a red '*' next to each required field

**Review by bsuro10 — APPROVED** [review]

**Pull request merged**